core/src/main/java/com/novoda/noplayer/SubtitleView.java

```diff
@@ -0,0 +1,82 @@
+package com.novoda.noplayer;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.util.AttributeSet;
+import android.view.View;
+
+import com.novoda.noplayer.model.NoPlayerCue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class SubtitleView extends View {
```

Comment:
I'm a little confused, we're not coupled to the `SubtitleView` in any way. In order for a client to provide their own subtitle displaying view they can implement `PlayerView` and that has a callback `setCues(List<NoPlayerCue> cues)`, these are the raw subtitles and any view could be used